### ISSUE TITLE
en_core_web_lg can't encode empty string (tested spacy 3.0, 3.2)

### ISSUE THREAD

**Issue by Fengjun-Wang** [explosion/spaCy#10334]
## How to reproduce the behaviour

```
import spacy
nlp = spacy.load('en_core_web_lg')
nlp("some text") # this one works
nlp("")  # empty string doesn't work. 
``` 

Error message example: 
```
  File "thinc/backends/numpy_ops.pyx", line 93, in thinc.backends.numpy_ops.NumpyOps.gemm
  File "blis/py.pyx", line 74, in blis.py.gemm
ValueError: Shape mismatch for blis.gemm: (0, 0), (576, 288)
```
You can reproduce the error to inspect more detailed error log. 

Note: spacy en_core_web_sm doesn't have this error. I believe this is an unexpected behavior, because ```sm``` model works well. It could be related to other py package np, blis etc., if you deep debug it. 

## Your Environment

* Operating System:  macOS Monterey | 12.2.1. (note: I tested on a Ubuntu 20.04.3 laptop, and reproduced same error)
* Python Version Used: 3.7.7 (note: 3.7.11 also has the error, so I am suspecting py 3.7 all have same error)
* spaCy Version Used: 3.0.3 (note: 3.2.0 also has the error)
* Environment Information: 
I use anaconda3
```
conda create --name spacyDebug python==3.7.7
conda activate spacyDebug
pip install spacy==3.0.3
python -m spacy download en_core_web_lg
```

**Comment by adrianeboyd** [comment]
Thanks for the report! This is related to #10324.

There's an actual bug in `spacy` to fix here that was masked by incorrect behavior in `blis`, but in the meanwhile you can downgrade to `blis==0.7.5` to avoid this error. We may yank this release of `blis` until this is fixed...

**Comment by Fengjun-Wang** [comment]
> 

You are right. I was using blis version 0.7.6. After downgrade to 0.7.5, it works. 
Thanks for your contribution and hope it will be fixed soon

**Comment by arnodunstatter** [comment]
I'm experiencing the same issue with blis 0.7.5

**Comment by adrianeboyd** [comment]
Okay, this should be fixed in the patch releases v3.0.8, v3.1.5, and v3.2.3. We will run a few more tests and then unyank blis v0.7.6 soon.
